assets/src/edit-story/components/panels/link/karma/linkPanel.karma.js

```diff
@@ -308,10 +308,10 @@ describe('Link Panel', () => {
       // Verify remove button exists
       expect(linkPanel.addressClear).toBeTruthy();
 
-      // Force focus to input - has to move out and back for weird reasons
       linkPanel.address.focus();
-      linkPanel.addressClear.focus();
+      expect(document.activeElement).toEqual(linkPanel.address);
 
+      await fixture.events.sleep(2000);
```

Comment:
That's quite a long wait -- what's it for?

Reply:
The test kept failing without it, not sure what it was waiting for.  I'll see if I can reduce it in length.

Reply:
Perhaps `waitFor` could be used here instead once it's clear what it's waiting for.

Reply:
Good call, I'll investigate!

Reply:
After reverting the changes to `designPanel.js`, this test didn't need any changes, so I reverted these changes tooo.